
{'date': 'Wed Apr 03 2019 18:10:58.300 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 9
    historia()
  module <module> line 6
    	cenaNinja = Cena(img = "https://i1.wp.com/homensdacasa.net/wp-content/uploads/2017/08/leitor-ninja-casa-lara-84.jpg?resize=960%2C642")
TypeError: 'module' object is not callable
'''},
{'date': 'Wed Apr 03 2019 18:26:19.660 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 7
  ninja = Elemento (img = elemento_ninja, 
                                                                                                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Apr 03 2019 18:29:28.541 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 6
  cenahouse = Cena(img = "http://farm3.static.flickr.com/2290/1888495823_ef459b5c3e.jpg
                                                                                       ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Wed Apr 03 2019 18:38:58.98 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 6
  cenaNinja = Cena(img = ""
                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Apr 03 2019 18:39:33.754 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 6
  cenaNinja = Cena(img = "http://farm3.static.flickr.com/2290/1888495823_ef459b5c3e.jpg"
                                                                                           ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Apr 03 2019 18:41:32.738 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 6
  cenaHouse= Cena(img = "http://farm3.static.flickr.com/2290/1888495823_ef459b5c3e.jpg"
                                                                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Apr 03 2019 18:53:15.837 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 6
  cenaHouse= Cena(img = "http://farm3.static.flickr.com/2290/1888495823_ef459b5c3e.jpg"
                                                                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Apr 03 2019 18:55:36.204 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 7
  ninja = Elemento (img=elemento_ninja, 
                                                                                                                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Apr 03 2019 18:56:31.441 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 12
    Historia()
NameError: name 'Historia' is not defined
'''},
{'date': 'Wed Apr 03 2019 18:56:31.907 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 12
    Historia()
NameError: name 'Historia' is not defined
'''},
{'date': 'Wed Apr 03 2019 18:56:32.336 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 12
    Historia()
NameError: name 'Historia' is not defined
'''},